editor/gl/program.ts

```diff
@@ -16,6 +16,13 @@ export class Program {
         let vertex_shader = gl.createShader(gl.VERTEX_SHADER);
         let fragment_shader = gl.createShader(gl.FRAGMENT_SHADER);
         let program = gl.createProgram();
+
+        // Make sure program is not null
+        // Removing that check make the compilation fail \o/
```

Comment:
Do you have any idea why this particular behavior occures? Is it because this script is executed by the compiler rather than compiled to javascript? Or is it because tsc is smart and reports an error here?